README.md

```diff
@@ -1,2 +1,126 @@
 # tackle2-ui
 Tackle (2nd generation) UI component.
+
+To install a Tackle2 cluster environment please refer to [Tackle documentation](https://github.com/konveyor/tackle).
+# Development
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/) >= 16.x
+## Getting Started 
+
+```
+git clone https://github.com/konveyor/tackle2-ui
+cd tackle-ui
+npm install
+```
+
+With a Tackle2 environment available (with kubectl authentication validated)  
+then one can start a tackle-ui instance locallly serving the pages from the current source code: 
+
+`npm run start:dev:local`
```

Comment:
need to specify which dir to run this in.

Reply:
Right we need that until we have npm workspace (hopefully soon) in place.